doc/releases.md

```diff
@@ -1,5 +1,52 @@
 # Releases
 
+## Version 1.20
+
+### Version 1.20.0
+
+**November 1, 2024**
+
+This release brings several enhancements and bug fixes.
```

Comment:
Would expand a little bit here, focusing on performance improvements, improved support for streaming and popup improvements.

Reply:
Are you sure we don't want to sneak in one more improved/improvements in that sentence? :upside_down_face: 

What about:
```suggestion
This release focuses on performance improvement, enhancing streaming support, and refining popup functionality, along with several bug fixes.
```